.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+matrix:
```

Comment:
I copied the matrix from the [tus-js-client](https://github.com/tus/tus-js-client), as it's our only dependency I figured we could use the same. (And maybe add a `latest` too).